### DISCUSSION TITLE
Great game! Lets WEB3 it!

### DISCUSSION THREAD

**Discussion by GenesisL1** [nickyvanurk/voidfall#27] [Ideas]
Hello!3
Your game is incredible! How about:

1. Make web3 auth via metamask
2. Give ability to mint (generate) NFT pictures with game and results
3. Overall "add" it to blockchain and allow some blockchain logics, for example pay to play
Of course best to see it @ genesisL1 blockchain - its fully Ethereum compatible:
genesisL1.com
Thank you!

**Comment by Browhatthe** [comment]
I thought they had said it would stay free. And NFTs? I feel like that's too much.
